pkg/apis/config/v1alpha1/kwok_configuration_types.go

```diff
@@ -153,6 +153,10 @@ type KwokConfigurationOptions struct {
 	// PodsOnNodeSyncParallelism is the number of workers to sync pods on nodes in parallel.
 	// +default=1
 	PodsOnNodeSyncParallelism uint `json:"podsOnNodeSyncParallelism,omitempty"`
+
+	// EnablePodsOnNodeSyncWatchList enables watch list for workers to sync pods on nodes.
+	// +default=false
+	EnablePodsOnNodeSyncWatchList *bool `json:"enablePodsOnNodeSyncWatchList"`
 }
```

Comment:
I think would it be better to change it to EnablePodsOnNodeSyncNoListPager=false or EnablePodsOnNodeSyncListPager=true